Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/Instance/InstanceSerializer.cpp

```diff
@@ -23,6 +23,53 @@ namespace AzToolsFramework
     {
         AZ_CLASS_ALLOCATOR_IMPL(JsonInstanceSerializer, AZ::SystemAllocator, 0);
 
+        namespace Internal
+        {
+            inline static const char* PathStartingWithEntities = "/Entities/";
+            inline static const char* PathMatchingContainerEntity = "/ContainerEntity";
+
+            //! Identifies the instance members to reload by parsing through the patches provided.
+            static void IdentifyInstanceMembersToReload(
+                PrefabDom& patches, AZStd::unordered_set<EntityAlias>& entitiesToReload, bool& shouldReloadContainerEntity)
+            {
+                for (const PrefabDomValue& patchEntry : patches.GetArray())
+                {
+                    PrefabDomValue::ConstMemberIterator patchEntryIterator = patchEntry.FindMember("path");
+                    if (patchEntryIterator != patchEntry.MemberEnd())
+                    {
+                        AZStd::string patchPath = patchEntryIterator->value.GetString();
+                        if (patchPath.starts_with(PathStartingWithEntities))
+                        {
+                            patchPath.erase(0, strlen(PathStartingWithEntities));
```

Comment:
This should use an AZStd::string_view for efficiency here to avoid making a copy Dom member value
A string_view also allows for the use of a quick remove_prefix operation where all it has to do is increment the begin view address by the number of characters in the string "/Entities"

```suggestion
                        AZStd::string_view patchPath = patchEntryIterator->value.GetString();
                        if (patchPath.starts_with(PathStartingWithEntities))
                        {
                            patchPath.remove_prefix(PathStartingWithEntities.size());
```